test/screenshot/lib/git-repo.js

```diff
@@ -23,18 +23,35 @@ class GitRepo {
     this.repo_ = simpleGit(workingDirPath);
   }
 
+  /**
+   * @return {!Promise<void>}
+   */
+  async fetch() {
+    return this.repo_.fetch();
```

Comment:
Can we confirm this downloads the shallowest version of the repo possible?

Reply:
Good call! I added these args to the `git fetch` call: `--depth=1 origin master`

So it should only fetch the tip commit of the `master` branch, and nothing else.

Does that seem sufficient?

Reply:
Yaaaaaaa! Nice!